input/core/src/commonMain/kotlin/majestic/Tag.kt

```diff
@@ -18,19 +19,22 @@ import androidx.compose.ui.unit.sp
 fun Tag(
     label: String,
     fontSize: TextUnit = 12.sp,
-    color: Color = Color(0xFF64B5F6),
-    backgroundColor: Color = Color(0xFF5C6BC0).copy(alpha = 0.2f)
+    color: Color = Color(0xFF66BB6A),
+    modifier: Modifier = Modifier
+        .clip(RoundedCornerShape(5.dp))
+        .background(Color(0xFF5C6BC0).copy(alpha = 0.2f))
```

Comment:
Use the newly created Modifier.tag() as the default value

```suggestion
    modifier: Modifier = Modifier.tag()
```